[16.0][OU-ADD] purchase_requisition: safety remove no update xml record

SonCrits opened the pull request:
safety remove no update xml record

royle-vietnam committed:
[IMP] openupgrade_framework: tracking number of rows per table before
and after the migration

remytms committed:
[OU-ADD] base_iban: Nothing to do

remytms committed:
[OU-ADD] base_vat: Nothing to do

remytms committed:
[OU-ADD] hr_contract

Most of the changes are managed by the migration script of hr module.
Load the noupdate_changes file.

nguyenvietlam0640 committed:
[MIG] hr_attendance: migrate 15 to 16

duyquyen96 committed:
[OU-ADD] survey: Nothing to do

SonCrits committed:
[OU-ADD] project_hr_expense: Nothing to do

SonCrits committed:
[OA-ADD] project_mrp: Nothing to do

SonCrits committed:
[OA-ADD] project_purchase: Nothing to do

nguyenvietlam0640 committed:
[MIG] fleet: migrate 15 to 16

nguyenvietlam0640 committed:
[MIG] maintenance: migrate 15 to 16

nguyenvietlam0640 committed:
[MIG] hr_maintenance: migrate 15 to 16

nguyenvietlam0640 committed:
[MIG] hr_holidays_attendance: migrate 15 to 16

SonCrits committed:
[OU-ADD] project_sms: Nothing to do

SonCrits committed:
[OU-ADD] project_timesheet_holidays: Nothing to do

nguyenvietlam0640 committed:
[MIG] hr_skills_survey: Migrate 15 to 16

nguyenvietlam0640 committed:
[MIG] hr_org_chart: Migrate 15 to 16

hoangtiendung070797 committed:
[OU-ADD] stock_sms: nothing to do

SonCrits committed:
[OU-ADD] purchase_requisition: migration data

- migration analytic_distribution
- join purchase order with purchase order group
- remove safety noupdate xml record

legalsylvain commented:
/ocabot migration purchase_requisition

<dependency>Depends on : 

- purchase : #4303</dependency>

nguyenvietlam0640 committed:
[MIG] hr_gamification: migrate 15 to 16

SonCrits committed:
[OU-ADD] project: migration data

locdang8 committed:
[OU-ADD] website_links: nothing to do

nguyenvietlam0640 committed:
[MIG] hr_recruitment: Migrate 15 to 16

nguyenvietlam0640 committed:
[MIG] hr_recruitment_survey: Migrate 15 to 16

royle-vietnam committed:
[OU-ADD] sale: migration

SonCrits committed:
[OA-ADD] auth_totp_mail: Nothing to do

SonCrits committed:
[OA-ADD] membership: Nothing to do

daiduongnguyen-odoo committed:
[OU-ADD] google_gmail: Migration done

duyquyen96 committed:
[OU-ADD] website_mass_mailing: Nothing to do

hoangtiendung070797 committed:
[OU-ADD] sale_timesheet: migrate 15.0 to 16.0

duyquyen96 committed:
[OU-ADD] website_mail: Nothing to do

duyquyen96 committed:
[OU-ADD] website_mail_group: Nothing to do

duyquyen96 committed:
[OU-ADD] website_membership: Nothing to do

duyquyen96 committed:
[OU-ADD] website_partner: Nothing to do

locdang8 committed:
[OU-ADD] gamification_sale_crm: nothing to do

locdang8 committed:
[OU-ADD] web_unsplash: nothing to do

locdang8 committed:
[OU-ADD] sms: nothing to do

SonCrits committed:
[OU-ADD] calendar_sms: Nothing to do

SonCrits committed:
[OU-ADD] product_matrix: Nothing to do

SonCrits committed:
[OA-ADD] product_margin: Nothing to do

daiduongnguyen-odoo committed:
[OU-ADD] analytic: Migration done

Need test with account also

daiduongnguyen-odoo committed:
[OU-ADD] website_google_map: Nothing to do

nguyenvietlam0640 committed:
[MIG] hr_timesheet_attendance: migrate 15 to 16

nguyenvietlam0640 committed:
[MIG] hr_presence: migrate 15to 16

locdang8 committed:
[OU-ADD] mass_mailing: migration

nguyenvietlam0640 committed:
[MIG] hr_skills_slides: migrate 15 to 16

daiduongnguyen-odoo committed:
[IMP] update analysis files

NgoHoangHung committed:
[OU ADD] crm_sms

NgoHoangHung committed:
[OU ADD] crm_mail_plugin

vietphuong97 committed:
Merge pull request #350 from duong77476/v16_imp_update_analyst_file

[IMP [16.0] *: update analysis files

NgoHoangHung committed:
[OU-ADD] website_event_questions: migration

hoangtiendung070797 committed:
[OU-ADD] stock_landed_costs: nothing to do

locdang8 committed:
[OU-ADD] website_blog: nothing to do

hoangtiendung070797 commented on openupgrade_scripts/scripts/purchase_requisition/16.0.0.1/upgrade_analysis_work.txt:
```diff
@@ -0,0 +1,41 @@
+# NOTHING TO DO
+
+---Fields in module 'purchase_requisition'---
+purchase_requisition / product.template         / purchase_requisition (selection): DEL selection_keys: ['rfq', 'tenders']
```
@SonCrits I found 1 commit https://github.com/odoo/odoo/commit/00689e7fc61ac08a2f75d856dc077a9ef46b2b26 to remove Call to Tender form. I think you need to migrate old data to Blanket Order. Maybe I didn't understand correctly, please check again

locdang8 committed:
[OU-ADD] website_jitsi: nothing to do

hoangtiendung070797 committed:
[OU-ADD] sale_stock_margin: nothing to do

hoangtiendung070797 committed:
[OU-ADD] sale_timesheet_margin: nothing to do

duyquyen96 committed:
[OA-ADD] website_sale_wishlist: Nothing to do

duyquyen96 committed:
[OA-ADD] website_event_track_quiz: Nothing to do

duyquyen96 committed:
[OA-ADD] website_event_meet_quiz: Nothing to do

duyquyen96 committed:
[OA-ADD] website_event_track_live: Nothing to do

duyquyen96 committed:
[OA-ADD] website_event_meet: Nothing to do

duyquyen96 committed:
[OA-ADD] website_slides_forum: Nothing to do

diamondnguyenkc committed:
[OU-ADD] event_booth: Nothing to do

diamondnguyenkc committed:
[OU-ADD] event_sms: Nothing to do

diamondnguyenkc committed:
[OU-ADD] event_sale: Nothing to do

nguyenvietlam0640 committed:
[MIG] website_hr_recruitment: migrate 15 to 16

diamondnguyenkc committed:
[OU-ADD] sale_crm: Nothing to do

diamondnguyenkc committed:
[OU-ADD] sale_product_matrix: Nothing to do

diamondnguyenkc committed:
[OU-ADD] sale_margin: Nothing to do

diamondnguyenkc committed:
[OU-ADD] sale_expense: Migrate

diamondnguyenkc committed:
[OU-ADD] sale_project_stock: Nothing to do

diamondnguyenkc committed:
[OU-ADD] sale_expense_margin: Nothing to do

diamondnguyenkc committed:
[OU-ADD] sale_product_configurator: Nothing to do

nguyenvietlam0640 committed:
[MIG] website_event: migrate 15 to 16

locdang8 committed:
[OU-ADD] website_crm: nothing to do

locdang8 committed:
[OU-ADD] website_livechat: nothing to do

locdang8 committed:
[OU-ADD] website_crm_livechat: nothing to do

NgoHoangHung committed:
[OU-ADD] website_sale_stock: migrate 15 to 16

nguyenvietlam0640 committed:
[MIG] website_event_track: migrate 15 to 16

nguyenvietlam0640 committed:
[MIG] website_forum: migrate 15 to 16

nguyenvietlam0640 committed:
[MIG] website_event_sale: migrate 15 to 16

royle-vietnam committed:
Merge remote-tracking branch 'OCA/16.0' into 15.0

royle-vietnam committed:
Merge pull request #388 from SonCrits/merge_upstream_1

Merge upstream 16 230710 01

daiduongnguyen-odoo committed:
[MISC] apriori: add section for Viindoo (#399)

* [MISC] apriori: add section for Viindoo

* [MISC] apriori: add project section for Viindoo

---------

Co-authored-by: Tran Truong Son <truongson290893@gmail.com>

diamondnguyenkc committed:
[OU-ADD] event_booth_sale: Nothing to do (#383)

* [OU-ADD] event_booth_sale: Nothing to do

* Update openupgrade_scripts/scripts/event_booth_sale/16.0.1.1/upgrade_analysis_work.txt

---------

Co-authored-by: Roy Le <43790414+royleviindoo@users.noreply.github.com>

royle-vietnam committed:
[MISC] *: add upgrade_analysis_work

nguyenvietlam0640 committed:
[OU-ADD] hr_hourly_cost: nothing todo (#408)

* [MIG] hr_hourly_cost: migrate 15 to 16

* Update openupgrade_scripts/scripts/hr_hourly_cost/16.0.1.0/upgrade_analysis_work.txt

---------

Co-authored-by: Roy Le <43790414+royleviindoo@users.noreply.github.com>

SonCrits committed:
[IMP] apriori: merge and renamed module (#442)


---------

Co-authored-by: quyen <duyquyencnt55@gmail.com>

nguyenvietlam0640 committed:
[MIG] account_fleet: migrate 15 to 16

hoangtiendung070797 committed:
[OU-ADD] stock_account: nothing to do (#356)

* [OU-ADD] stock_account: nothing to do

* Update openupgrade_scripts/scripts/stock_account/16.0.1.1/upgrade_analysis_work.txt

---------

Co-authored-by: Roy Le <43790414+royleviindoo@users.noreply.github.com>

nguyenvietlam0640 committed:
[IMP] apriori: add more scripts

nguyenvietlam0640 committed:
[FIX] apriori: fix duplicate

NgoHoangHung committed:
[OU-ADD] attachment indexation: Nothing to do

nguyenvietlam0640 committed:
[MIG] mass_mailing_crm: migrate 15 to 16

nguyenvietlam0640 committed:
[MIG] mass_mailing_sale: migrate 15 to 16

nguyenvietlam0640 committed:
[MIG] mass_mailing_event: migrate 15 to 16

nguyenvietlam0640 committed:
[MIG] mass_mailing_slides: migrate 15 to 16

nguyenvietlam0640 committed:
[MIG] mass_mailing_event_track: migrate 15 to 16

daiduongnguyen-odoo committed:
[FIX] apriori: wrong section for 'to_hr_contract_actions'

NgoHoangHung committed:
[OU-ADD] website_profile: migration (#453)

* [OU-ADD] website_profile: Nothing to do

* Update docsource/modules150-160.rst

---------

Co-authored-by: Roy Le <43790414+royleviindoo@users.noreply.github.com>

NgoHoangHung committed:
[OU-ADD] account_edi_ubl: Nothing to do (#452)

* [OU-ADD] account_edi_ubl: Nothing to do

* Update modules150-160.rst

---------

Co-authored-by: Roy Le <43790414+royleviindoo@users.noreply.github.com>

NgoHoangHung committed:
[OU-ADD] account_sale_timesheet: Nothing to do (#449)

* [OU-ADD] account_sale_timesheet: Nothing to do

* Update modules150-160.rst

---------

Co-authored-by: Roy Le <43790414+royleviindoo@users.noreply.github.com>

locdang8 committed:
[OU-ADD] website_crm_iap_reveal: nothing to do

daiduongnguyen-odoo committed:
[FIX] mail: create constrain to avoid migration error when install
'to_mail_notif_and_email' module

So in the mail module v16, it has a constrain call
unique_mail_message_id_res_partner_id_if_set, but in the module
'to_mail_notif_and_email' we drop that constains and create a new with
additional condition so when migrating from 15 to 16 it will get error
when registry try to create the constaint.
To avoid that, we should check if the module 'to_mail_notif_and_email'
is installed then instanly create constrain index specify in that module

daiduongnguyen-odoo committed:
[FIX] apriori: wrong merge module

"viin_website_image_optimization_disable": "viin_web_editor",

pedrobaeza commented:
Please don't rebase over an outdated branch, as the pull request got contaminated with hundred of commits.

SonCrits committed:
[FIX] apriori: rename module before merge

diamondnguyenkc committed:
[IMP] apriori: rename module

hoangtiendung070797 committed:
[OU-ADD] stock: migration (#466)

* [OU-ADD] stock: migration

* imp

---------

Co-authored-by: Roy Le <roy.le@viindoo.com>

ngochuy97hp committed:
[OU-ADD] sale_quotation_builder: mig to 16.0

ngochuy97hp committed:
[OU-ADD] sale_management: mig to 16.0

ngochuy97hp committed:
[OU-ADD] sale_purchase: mig to 16.0

royle-vietnam committed:
[MIG] sale_project: nothing todo

diamondnguyenkc committed:
[IMP] apriori: Merged module

ngochuy97hp committed:
[OU-ADD] l10n_multilang: nothing to do

ngochuy97hp committed:
[OU-ADD] snailmail_account: mig to 16.0

ngochuy97hp committed:
[OU-ADD] event_crm_sale: nothing to do

SonCrits commented:
> Please don't rebase over an outdated branch, as the pull request got contaminated with hundred of commits.


We are currently in the final stages of migrating to version 16.0. However, we also welcome feedback from the community. We apologize for any inconvenience caused.
cc: @pedrobaeza

ngochuy97hp committed:
[MIG] account_edi: nothing to do

ngochuy97hp committed:
[OU-ADD] payment_paypal: mig to 16.0 (#478)

* [OU-ADD] payment_paypal: mig to 16.0

* Update openupgrade_scripts/scripts/payment_paypal/16.0.2.0/post-migration.py

---------

Co-authored-by: Roy Le <43790414+royleviindoo@users.noreply.github.com>

duyquyen96 committed:
[MIG] website: mig 15 to 16 (#375)

* [MIG] website: migrate 15 to 16

* [FIX] website: fix migrate

* [OU-ADD] website: BS4 to BS5 migration

* [FIX] website: Load the noupdate_changes

* [FIX] website: fix by comment

* Update openupgrade_scripts/scripts/website/16.0.1.0/pre-migration.py

* Update openupgrade_scripts/scripts/website/16.0.1.0/pre-migration.py

---------

Co-authored-by: nguyenvietlam0640 <lamnv0640@gmail.com>
Co-authored-by: duongnguyen <daiduongnguyen2709@gmail.com>
Co-authored-by: Roy Le <43790414+royleviindoo@users.noreply.github.com>

SonCrits committed:
[OA-ADD] purchase: Migration analytic account (#382)

* [OA-ADD] purchase: migration analytic distribution

* Update openupgrade_scripts/scripts/purchase/16.0.1.2/upgrade_analysis_work.txt

---------

Co-authored-by: Roy Le <43790414+royleviindoo@users.noreply.github.com>

hoangtiendung070797 committed:
[OU-ADD] purchase_stock: migration (#353)

* [OU-ADD] purchase_stock: migration

* Update openupgrade_scripts/scripts/purchase_stock/16.0.1.2/pre-migration.py

* Update openupgrade_scripts/scripts/purchase_stock/16.0.1.2/pre-migration.py

---------

Co-authored-by: Roy Le <43790414+royleviindoo@users.noreply.github.com>

ngochuy97hp committed:
[OU-ADD] website_sale_slides: mig to 16.0

royle-vietnam committed:
[MISC] *: fix workflow due to unfinished dependencies

royle-vietnam committed:
[MISC] *: fix workflow

hoangtiendung070797 committed:
[OU-ADD] purchase_requisition_stock: Nothing to do (#361)

* [OU-ADD] purchase_requisition_stock: Nothing to do

* Update openupgrade_scripts/scripts/purchase_requisition_stock/16.0.1.2/upgrade_analysis_work.txt

---------

Co-authored-by: Roy Le <43790414+royleviindoo@users.noreply.github.com>

daiduongnguyen-odoo committed:
[OU-ADD] hr: migrarion done

diamondnguyenkc committed:
[IMP] apriori: Renamed modules

nguyenvietlam0640 committed:
[OU-ADD] hr_skills (#403)

* [MIG] hr_skills: migrate 15 to 16

* Update openupgrade_scripts/scripts/hr_skills/16.0.1.0/pre-migration.py

---------

Co-authored-by: Roy Le <43790414+royleviindoo@users.noreply.github.com>

nguyenvietlam0640 committed:
[OU-ADD] hr_timesheet (#414)

* [MIG] hr_timesheet: migrate 15 to 16

* Update openupgrade_scripts/scripts/hr_timesheet/16.0.1.0/upgrade_analysis_work.txt

---------

Co-authored-by: Roy Le <43790414+royleviindoo@users.noreply.github.com>

Commit:
[IMP] apriori: add merged module

daiduongnguyen-odoo committed:
[MISC] apriori: google_drive, google_spreadsheet case

google_drive, google_spreadsheet is orinally from odoo but has been
remove since 16.0
because of some technical problem but they have been brought back in
viindoo yeahh

anhbtit committed:
[IMP] openupgrade_scripts: Add merged and renamed modules.

ngochuy97hp committed:
[OU-ADD] hr_expense: mig to 16.0

nguyenvietlam0640 committed:
[OU-ADD] hr_fleet (#401)

* [MIG] hr_fleet: migrate 15 to 16

* [FIX] hr_fleet: add pre-migration to create value for compute field with
store=true by sql instead of auto orm

* Update openupgrade_scripts/scripts/hr_fleet/16.0.1.0/pre-migration.py

---------

Co-authored-by: Roy Le <43790414+royleviindoo@users.noreply.github.com>

ngochuy97hp committed:
[OU-ADD] website_sale: mig to 16.0 (#482)

* [OU-ADD] website_sale: mig to 16.0

* Update openupgrade_scripts/scripts/website_sale/16.0.1.1/upgrade_analysis_work.txt

---------

Co-authored-by: Roy Le <43790414+royleviindoo@users.noreply.github.com>
